.updatecli/updatecli.d/bump-rancher-dep.yaml

```diff
@@ -0,0 +1,137 @@
+name: 'Bump EKS operator'
+
+scms:
+  rancher:
+    kind: github
+    spec:
+      user: highlander-ci-bot
```

Comment:
should we align all rancher bump flows using the same bot in all operators? currently it is only in [aks](https://github.com/rancher/aks-operator/blob/master/.updatecli/updatecli.d/bump-rancher-dep.yaml#L7) operator but could be in gke operator in the future

Reply:
Yes, the bot didn't exist when aks PR was created. I will update the config there too.